client/src/infographic/Infographic.yaml

```diff
@@ -61,6 +61,26 @@ infograph_explorer_link:
   en: Advanced search
   fr: Recherche avancée
 
+invalid_level_redirect_home:
+  transform: [handlebars]
+  en: The "{{level}}" is invalid. We redirected you to homepage.
+  fr: TODO
+invalid_subject_redirect_parent_dept:
+  transform: [handlebars]
+  en: The organization "{{subject_id}}" is invalid. We redirected you to its parent department.
+  fr: TODO
+invalid_subject_redirect_home:
+  transform: [handlebars]
+  en: The organization "{{subject_id}}" is invalid. We redirected you to homepage.
+  fr: TODO
+invalid_panel:
+  transform: [handlebars]
+  en: The link to panel is invalid.
```

Comment:
This doesn't need to be a redirect thing, right? The `JSURL` parsing in `componentDidUpdate` should handle errors itself (and probably fail quietly).

Reply:
Yes, if actual panel string is invalid. But if the whole structure is wrong, like `#orgs/gov/gov/infograph/financial/.-ERROR.-(panel_key.-.-'welcome_mat)`. It'll throw an error

Reply:
I meant the actual errors thrown by `JSURL` on unparsable URLs, yeah. If those options can't be parsed, we don't actually know if they were a `panel_key` or something else (we use them for like linking to indicator modals too, and generally the idea was that panels could use them for arbitrary state if that ever became necessary). Figure those errors should just be suppressed (and maybe logged to analytics), but don't really need to be communicated up to the user.